### DISCUSSION TITLE
Load char or stage in zip format

### DISCUSSION THREAD

**Discussion by leonkasovan** [ikemen-engine/Ikemen-GO#1929] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

In order to simplify the distribution of character or stage, we can use zip format. So we don't have to uncompress it first and we can save space. 

### Describe the solution you'd like

Just put zipped file in folder `chars` or `stages`.  
Then add zipped filename in file `select.def` in section [Characters] or [ExtraStages]
```ini
[Characters]
kfmZ, stages/kfm.def
kfm720, stages/kfm.def, order=0
;zipped chars
Ken.zip, random
Ryu.zip, random

[ExtraStages]
stages/stage1.def
stages/kfm.def
;zipped stages
stages/stage2.zip
stages/stage3.zip
```

### Search terms

zip compress chars stages char stage load compressed

**Comment by potsmugen** [comment]
This comes up every now and then and usually the consensus is that the benefit of saving on storage is cancelled by the increased loading times. Especially when storage has gotten so cheap.

**Reply by ZyonZZ** [reply]
and the loading time is eliminated with high capacity SSD or M2... so compressed files become a benefit..
it should be an option for those who require it

**Comment by K4thos** [comment, marked as answer]
implemented #2473
